pkgs/os-specific/bsd/netbsd/default.nix

```diff
@@ -453,39 +447,35 @@ let
     path = "include";
     version = "8.0";
     sha256 = "128m77k16i7frvk8kifhmxzk7a37m7z1s0bbmja3ywga6sx6v6sq";
-    nativeBuildInputs = [
+    nativeBuildInputs = with buildPackages.netbsd; [
       bsdSetupHook netbsdSetupHook
       makeMinimal
       install mandoc groff nbperf rpcgen
     ];
-    # Hack around coreutils from wrapper clobbering some things
-    preIncludes = ''
-      PATH="${buildPackages.netbsd.install}/bin:$PATH"
-      PATH="${buildPackages.netbsd.rpcgen}/bin:$PATH"
-    '';
-    extraPaths = [ common.src ];
+    extraPaths = with netbsd; [ common.src ];
     headersOnly = true;
     noCC = true;
-    # meta.platforms = lib.platforms.netbsd;
-    makeFlags = [ "RPCGEN_CPP=${buildPackages.gcc-unwrapped}/bin/cpp" ];
+    meta.platforms = lib.platforms.netbsd;
+    makeFlags = [ "RPCGEN_CPP=${buildPackages.stdenv.cc.cc}/bin/cpp" ];
```

Comment:
Wouldn't just `stdenv` be sufficient or correcter here? Or do buildPackages.stdenv.cc.cc and stdenv.cc.cc have the same effective hostPlatform they run on?

Reply:
At this point we don't yet have a CPP for the right stage, so we awkwardly use the previous stage's.

Reply:
I see, thanks for the clarification.